Gemfile

```diff
@@ -1,20 +1,20 @@
 source "https://rubygems.org"
 
-gem "rails", "4.2.0"
-gem "pg"
-gem "sass-rails", "~> 5.0"
-gem "uglifier", ">= 1.3.0"
-gem "coffee-rails", "~> 4.1.0"
-gem "jquery-rails"
-gem "turbolinks"
-gem "jbuilder", "~> 2.0"
-gem "sdoc", "~> 0.4.0", group: :doc
-gem "unicorn"
-gem "bcrypt"
-gem "money-rails"
-gem "bootstrap-sass", "~> 3.2.0"
-gem "autoprefixer-rails"
-
+gem 'rails', '4.2.0'
+gem 'pg'
+gem 'sass-rails', '~> 5.0'
+gem 'uglifier', '>= 1.3.0'
+gem 'coffee-rails', '~> 4.1.0'
+gem 'jquery-rails'
+gem 'turbolinks'
+gem 'jbuilder', '~> 2.0'
+gem 'sdoc', '~> 0.4.0', group: :doc
+gem 'unicorn'
+gem 'bcrypt'
+gem 'money-rails'
+gem 'bootstrap-sass', '~> 3.2.0'
+gem 'autoprefixer-rails'
+gem 'cancancan'
```

Comment:
Did you make sure to merge in your master before pushing this? It should be double quotes. Let's try and use those throughout the app to standardize it.